index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Graffiti bootcamp - project 3</title>
+  <!-- dont forget to add a css file and link it here! -->
+  <link rel="stylesheet" href="style.css">
+  <!-- Google fonts-->
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans|Raleway:300,400" rel="stylesheet">
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
```

Comment:
Interesting. When including something like this, it can be very useful for the reader to know what elements/styles/etc are being used from the external resource.  This can be done:

- As a comment below this include
- As a comment at the usage instance of the style in your code